cypress.json

```diff
@@ -0,0 +1 @@
+{"baseUrl":"http://localhost:3003" }
```

Comment:
This configuration is required by Cypress to run cypress without setting the url to navigate on every test step. Once this variable is set here, the url can then be overwritten with the script to run the test suite. See the npm scripts for the tests in the root of the directory. 

https://docs.cypress.io/guides/references/configuration.html#Options